packages/core/src/components/menu/_common.scss

```diff
@@ -26,28 +36,67 @@ $dark-menu-background-color: $dark-gray4 !default;
   color: inherit;
   user-select: none;
 
-  &:not(#{$disabled-selector})#{$hover-selector} {
+  &#{$active-selector},
+  &#{$active-selector}#{$hover-pseudoselector} {
     background: $pt-intent-primary;
     cursor: pointer;
     color: $white;
   }
 
-  &#{$disabled-selector} {
+  &#{$disabled-selector},
+  &#{$disabled-selector}#{$hover-pseudoselector} {
+    // override global a:focus styles
+    outline: none;
+    background: inherit;
     cursor: not-allowed;
     color: $pt-text-color-disabled;
   }
 
+  &:not(#{$active-selector}):not(#{$disabled-selector}) {
```

Comment:
this is frightening. can you just put the pseudoclass bits first so active and disabled will override them simply by appearing later in the CSS?